deploy/floorist.yaml

```diff
@@ -0,0 +1,203 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: sources-floorist
+objects:
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: sources
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOG_LEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    queries:
+      # We ignore the tenants that belong to QE and developer accounts. More info on:
+      #
+      # - https://issues.redhat.com/browse/RHCLOUD-20850
+      # - https://lists.corp.redhat.com/archives/insights-platform/2022-September/000622.html
+      - prefix: insights/sources/all-sources-count
+        query: |
+          SELECT
+            count("src".*) AS "number_sources_all"
+          FROM
+            "sources" AS "src"
+          WHERE
+            "src"."tenant_id" NOT IN (2, 5, 7, 11, 14, 27, 31, 33, 15356, 15409, 24317, 24420, 24632, 39273)
```

Comment:
Those tenant IDs were created either by QE people or Red Hat developers that were trying to integrate with us. Kathryn told me to remove them so that the statistics don't include our internal tenants.

Reply:
Thanks for explanation @MikelAlejoBR !